api/envoy/config/overload/v3/overload.proto

```diff
@@ -136,12 +136,25 @@ message OverloadAction {
   // A set of triggers for this action. The state of the action is the maximum
   // state of all triggers, which can be scaling between 0 and 1 or saturated. Listeners
   // are notified when the overload action changes state.
+  // An overload manager action can only have one trigger for a given resource.
   repeated Trigger triggers = 2 [(validate.rules).repeated = {min_items: 1}];
 
   // Configuration for the action being instantiated.
   google.protobuf.Any typed_config = 3;
 }
 
+message LoadShedPoint {
```

Comment:
Please describe what is a Load-shed point.

Reply:
done.